### DISCUSSION TITLE
Pull-to-refresh not working in empty Notifications tab

### DISCUSSION THREAD

**Discussion by mickael-menu** [community/community#3796] [Mobile]
When in the empty "All caught up!" **Notifications** tab, **in the Android app**, we can't do a pull-to-refresh to get new notifications. The only way to see if there are new notifications is to:

* Keep at least one notification in the list, to be able to do a pull-to-refresh
* Kill and restart the app

**Comment by nplasterer** [comment]
Thanks for the feedback! We're looking into a fix for this now.
